### PR TITLE
feat(toggle-group): v2 design

### PR DESCRIPTION
**Pull request by ttalbot**
image

### CODE REVIEW

File: libs/web-components/button-toggle-group/package.json [hunk 1]
```diff
@@ -12,11 +12,10 @@
   "main": "dist/src/index.js",
   "module": "dist/src/index.js",
   "scripts": {
-    "build:style": "node ../../../scripts/sass-to-lit-css/index.js src/styles.scss src/button-toggle/styles.scss src/button-toggle/filter-styles.scss"
+    "build:style": "node ../../../scripts/sass-to-lit-css/index.js src/styles.scss src/button-toggle/styles.scss"
   },
   "dependencies": {
-    "@material/mwc-tab": "0.27.0",
-    "@material/mwc-tab-bar": "0.27.0",
+    "@finastra/icon": "1.2.9",
```
Comment (bcldvd) [inline libs/web-components/button-toggle-group/package.json:18]: 1.2.10

File: libs/web-components/button-toggle-group/src/button-toggle-group.ts [hunk 2]
```diff
@@ -1,25 +1,100 @@
...
+    }
+    
+    this.value = selectedButton.getAttribute('value') || selectedButton.getAttribute('label') || selectedButton.getAttribute('icon') || '';
+    
+    if (changedProperties.has('selectedIndex')) {
+      this._select(selectedButton);
+
+      requestAnimationFrame(() => {
+        const rect = selectedButton.getBoundingClientRect();
+        this.style.setProperty('--fds-toggle-selection-width', (rect.width) + 'px');
```
Comment (bcldvd) [inline libs/web-components/button-toggle-group/src/button-toggle-group.ts:59]: Could be documented

File: libs/web-components/button-toggle-group/src/button-toggle-group.ts [hunk 3]
```diff
@@ -1,25 +1,100 @@
...
+    }
+  }
+
+  private _select(button: ButtonToggle) {
+    const rect:DOMRect = button.getBoundingClientRect();
+
+    this._resetSelection();
+    button.classList.add('selected');
+    
+    this.style.setProperty('--fds-toggle-selection-x', (button.offsetLeft - 5) + 'px');
```
Comment (bcldvd) [inline libs/web-components/button-toggle-group/src/button-toggle-group.ts:77]: Could also be documented

File: libs/web-components/button-toggle-group/src/button-toggle/button-toggle.ts [hunk 4]
```diff
@@ -1,27 +1,67 @@
...
+  protected renderIcon() {
+    return html` <fds-icon> ${this.icon} </fds-icon>`;
+  }
+
+  protected renderLabel() {
+    return html` <span> ${this.label} </span>`;
+  }
+
+  private _handleClick() {
+    this.dispatchEvent(new CustomEvent('FDSToggle:click', { bubbles: true, composed: true }));
```
Comment (bcldvd) [inline libs/web-components/button-toggle-group/src/button-toggle/button-toggle.ts:63]: Could be documented

**Review by bcldvd — APPROVED** [review]

File: libs/web-components/button-toggle-group/stories/button-toggle-group.stories.ts [hunk 5]
```diff
@@ -2,59 +2,48 @@ const README = require('../README.md');
...
 
 export default {
   title: 'ACTIONS/Toggle',
   component: 'fds-button-toggle-group',
   argTypes,
   parameters: {
+    actions,
     docs: {
-      description: { component: allSanitizers(README) }
+      description: { component: README }
```
Comment (dorsaffrigui) [inline libs/web-components/button-toggle-group/stories/button-toggle-group.stories.ts:14]:
```suggestion
      description: { component: allSanitizers(README) }
```

**Review by dorsaffrigui — APPROVED** [review]

**Pull request merged**